Makefile

```diff
@@ -40,28 +29,16 @@ $(CROOT)/lexer.c $(CROOT)/lexer.h: $(CROOT)/lexer.l $(CROOT)/grammar.h
 	$(LEX) --outfile=$(CROOT)/lexer.c --header-file=$(CROOT)/lexer.h $<
 
 
-SOURCES =					\
-	$(CROOT)/decimal.c			\
-	$(CROOT)/decimal.h 			\
-	$(CROOT)/lexer.c			\
-	$(CROOT)/lexer.h			\
-	$(CROOT)/grammar.c			\
-	$(CROOT)/grammar.h			\
-	$(CROOT)/macros.h			\
-	$(CROOT)/tokens.h
-
 .PHONY: build
-build: $(SOURCES)
-	$(PYTHON) setup.py build_ext -i
-
-$(CROOT)/tokens_test: $(CROOT)/tokens_test.o $(CROOT)/tokens.o $(CROOT)/decimal.o
+build:
+	meson setup --reconfigure build/
```

Comment:
I'm all for modernization, but this does include a new dependency on meson though.
How reliable is that? Will it work on Windows? On macOS?

Reply:
Yes, it works on Windows and macOS. It is how the binary Python packages we publish are built and also how the package is build if someone installs it on a platform for which we don't publish a binary. I think it is more reliable than setuptools. The build system we use is the same used by numpy, scipy, pandas, matplotlib and others.